libs/components/indicators/src/lib/modules/label/label.component.ts

```diff
@@ -53,4 +109,39 @@ export class SkyLabelComponent {
     this.baseIcon = indicatorIcon.modernThemeBaseIcon;
     this.topIcon = indicatorIcon.modernThemeTopIcon;
   }
+
+  #updateDescriptionComputed(): void {
+    if (this.descriptionType) {
+      switch (this.descriptionType) {
+        case 'none':
+          this.descriptionComputed = undefined;
+          break;
+        case 'custom':
+          this.descriptionComputed = this.customDescription;
+          break;
+        default:
+          this.#unsubscribe();
```

Comment:
I'm going to move this to the top of this function so if they go from a defined type to custom or none for whatever reason this still gets unsubscribed